internal/resources/resource_host.go

```diff
@@ -220,6 +221,14 @@ func hostSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "map of label name to label value for this host",
 		},
+		hDiscoveryIPs: {
+			Type:     schema.TypeList,
+			Optional: true,
+			Elem: &schema.Schema{
```

Comment:
Since this is not user configurable, 'Computed: true' is the right flag here.
https://www.terraform.io/plugin/sdkv2/schemas/schema-behaviors#computed

Reply:
Change name to ISCSIDiscoveryIP matching quake and Computed: true